sdk/tests/conformance2/reading/read-pixels-pack-parameters.html

```diff
@@ -262,83 +267,83 @@
     debug("");
     var destText = usePixelPackBuffer ? "PIXEL_PACK buffer" : "array buffer";
     debug("Verify that reading pixels to " + destText + " works fine with various pack alignments.");
-    runTestIteration(0, 0, 1, 3, {alignment:1}, usePixelPackBuffer);
-    runTestIteration(0, 0, 1, 3, {alignment:2}, usePixelPackBuffer);
-    runTestIteration(0, 0, 1, 3, {alignment:4}, usePixelPackBuffer);
-    runTestIteration(0, 0, 1, 3, {alignment:8}, usePixelPackBuffer);
-    runTestIteration(0, 0, 2, 3, {alignment:4}, usePixelPackBuffer);
-    runTestIteration(0, 0, 2, 3, {alignment:8}, usePixelPackBuffer);
-    runTestIteration(0, 0, 3, 3, {alignment:4}, usePixelPackBuffer);
-    runTestIteration(0, 0, 3, 3, {alignment:8}, usePixelPackBuffer);
-    runTestIteration(0, 0, 0, 0, {alignment:1}, usePixelPackBuffer);
-    runTestIteration(0, 0, 1, 3, {alignment:4}, usePixelPackBuffer);
-    runTestIteration(0, 0, 1, 3, {alignment:8}, usePixelPackBuffer);
+    runTestIteration(0, 0, 1, 3, {alignment:1}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 1, 3, {alignment:2}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 1, 3, {alignment:4}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 1, 3, {alignment:8}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 2, 3, {alignment:4}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 2, 3, {alignment:8}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 3, 3, {alignment:4}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 3, 3, {alignment:8}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 0, 0, {alignment:1}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 1, 3, {alignment:4}, usePixelPackBuffer, true);
+    runTestIteration(0, 0, 1, 3, {alignment:8}, usePixelPackBuffer, true);
 
     debug("");
     debug("Verify that reading pixels to " + destText + " works fine with PACK_ROW_LENGTH < width.");
```

Comment:
Perhaps update the comment like:
Verify that reading pixels to [destText] is disallowed when PACK_ROW_LENGTH < width.
? If that is actually what is tested.